src/PriorityList/PHP81Implementation.php

```diff
@@ -0,0 +1,281 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\Stdlib\PriorityList;
+
+use Countable;
+use Exception;
+use Iterator;
+
+use function array_map;
+use function current;
+use function key;
+use function next;
+use function reset;
+use function uasort;
+
+class PHP81Implementation implements Iterator, Countable
```

Comment:
Should be `@internal` IMO: the implementation should not be part of the public API